samcli/commands/logs/command.py

```diff
@@ -163,9 +150,6 @@ def do_cli(
     from samcli.lib.utils.boto_utils import get_boto_client_provider_with_config, get_boto_resource_provider_with_config
 
     if not names or len(names) > 1:
```

Comment:
I think this if statement needs some adjustments right? It was either asking for beta feature opt in or printing an informational message.

Reply:
This if statement check if the name is not provided we will print the hint. In the previous beta version we check if the beta flag is off then we don't print the hint then exist but now we don't need the beta check anymore.

Reply:
But there was an `else` statement there which was checking;
- If no names or multiple names provided, ask for beta opt-in
- Else print informational message that `sam logs` can be used without any name flag.

So now it seems like it is printing the information message if no names or multiple names are provided. It should be opposite.

Reply:
I see. Thanks for the explanation. Updated the logic.